DOL.WHD.Section14c.Web/src/modules/components/dolHeader/dolHeaderController.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+module.exports = function(ngModule) {
+  ngModule.controller('dolHeaderController', function(
+    $scope,
+    $location,
+    $route,
+    $anchorScroll,
+    navService,
+    autoSaveService,
+    validationService
+  ) {
+    'ngInject';
+    'use strict';
+
+    $scope.skipToMainContent = function (id) {
```

Comment:
I appreciate that the element ID is an argument to this function and not hardcoded into the controller.  👍